### PR TITLE
Add NetBSD support

### PR DESCRIPTION
**Pull request by FGasper**

### CODE REVIEW

**Comment by saghul** [comment]
Looks like the way to get the number of cpus is different on netbsd...

**Comment by saghul** [comment]
Or you can wait until https://github.com/quickjs-ng/quickjs/pull/178 lands and it should take care of it...

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -416,3 +416,19 @@ jobs:
...
+        uses: vmactions/netbsd-vm@v1
+        with:
+          usesh: true
+          prepare: |
+            /usr/sbin/pkg_add cmake
+          run: |
+            cmake -B build
+            cmake --build build -j $(sysctl -n hw.ncpu)
+            ./build/qjs -qd
+            make test
```
Comment (bnoordhuis) [inline .github/workflows/ci.yml]:
```suggestion
            /usr/sbin/pkg_add cmake gmake
          run: |
            cmake -B build
            cmake --build build -j $(sysctl -n hw.ncpu)
            ./build/qjs -qd
            gmake test
```

**Comment by saghul** [comment]
If you rebase now, and install gmake, it should just work.

**Comment by FGasper** [comment]
@bnoordhuis Should be good to go.

**Review by saghul — APPROVED** [review]

**Pull request merged**